src/System Application/App/Edit in Excel/src/EditInExcelWorkbookImpl.Codeunit.al

```diff
@@ -231,12 +231,39 @@ codeunit 1489 "Edit in Excel Workbook Impl."
         exit(99);
     end;
 
+    local procedure UserSettingsLanguageID(): Integer
+    var
+        DefaultLanguageID: Integer;
+        UserSessionSettings: SessionSettings;
+    begin
+        if TryUserSettingsLanguageID() then begin
+            UserSessionSettings.Init();
+            exit(UserSessionSettings.LanguageId());
+        end;
+
+        if WindowsLanguage() <> 0 then
```

Comment:
You can always assign DefaultLanugageID with WindowsLanuguage() and then only overwrite it if it equals `0`.
That way you can avoid to call WindowsLanuguage() twice.